app/controllers/ClaimToAdjustPoaController.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import connectors.ClaimToAdjustPoaConnector
+import controllers.predicates.AuthenticationPredicate
+import models.claimToAdjustPoa.ClaimToAdjustPoaApiResponse._
+import models.claimToAdjustPoa.ClaimToAdjustPoaResponse.ErrorResponse
+import models.claimToAdjustPoa.ClaimToAdjustPoaRequest
+import play.api.{Logger, Logging}
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, ControllerComponents, Request, Result}
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ClaimToAdjustPoaController @Inject()(authentication: AuthenticationPredicate,
```

Comment:
:thinking: - we usually wrap connector with the service and call service from the connector, but I guess in this case there is no reason to create service at this moment, so this approach should be fine